### PR TITLE
Crafting and seen buttons animation

### PR DESCRIPTION
**Pull request by odeyah**
created the animation for both buttons with react-Spring

### CODE REVIEW

File: src/Screens/Gameplay/Components/Crafting/CraftingButton.jsx [hunk 1]
```diff
@@ -1,7 +1,35 @@
-import React from 'react';
+//import { useRef } from 'react';
```
Comment (elShailee) [inline src/Screens/Gameplay/Components/Crafting/CraftingButton.jsx:1]: please dont keep commented code

**Comment by elShailee** [comment]
image
for me - the crafting button gets cut of a tiny bit on the right while minimizing.
this is how it looks after the animation, it is more noticeable mid-animation.

it seems to be worse when I make the window shorter:
image
image

**Comment by odeyah** [comment]
> 
> 
> image
> for me - the crafting button gets cut of a tiny bit on the right while minimizing.
> this is how it looks after the animation, it is more noticeable mid-animation.
> 
> it seems to be worse when I make the window shorter:
> image
> image

changed the animation from 2vw to 4vh and now it works

**Review by elShailee — APPROVED** [review]
Melech!

**Pull request merged**